docs/sources/writing-guide/shortcodes/index.md

```diff
@@ -41,6 +41,34 @@ Kingston is the capital of Jamaica.
 {{%/* /admonition */%}}
 ```
 
+## `docs/link` shortcode
+
+The `docs/link` shortcode offers more flexible linking than the Hugo builtin `relref` shortcode.
+Each argument to the shortcode has the form `<PATH PREFIX> -> <REFERENCE>`.
+_`PATH PREFIX`_ is matched against the page's URL path.
+If _`PATH PREFIX`_ matches, then _`REFERENCE`_ is used as the argument to Hugo's `relref` shortcode.
+
+_`REFERENCE`_ can include a version variable that is looked up in the page's front matter.
```

Comment:
```suggestion
The `docs/link` shortcode offers more flexible linking than the Hugo builtin `relref` shortcode.
Each argument to the shortcode has the form `<PATH PREFIX> -> <REFERENCE>`:
* _`PATH PREFIX`_ matches against the page's URL path. If _`PATH PREFIX`_ matches, then _`REFERENCE`_ is used as the argument to Hugo's `relref` shortcode.
* _`REFERENCE`_ can include a version variable that is looked up in the page's front matter.
```

Reply:
Edited your suggestion to make it match `prettier` style.